docs/UserGuide.md

```diff
@@ -64,53 +64,65 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
+### Adding a person: `cadd`
 
-Format: `help`
+Adds a person to the contact list.
 
+Format: `cadd n/NAME e/EMAIL [p/PHONE_NUMBER] [a/ADDRESS] [th/TELEGRAM_HANDLE] [z/ZOOM] [t/TAG]…`
 
-### Adding a person: `add`
+* A person can have any number of tags (including 0)
 
-Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Examples:
+* `cadd n/Alex Doe e/e0123456@u.nus.edu a/COM1 #99-99 th/johnDoe99 t/Professor`
+* `cadd n/ Jon Cheng t/TA e/e7654321@u.nus.edu a/COM1-0201 p/87654321 t/Senior th/jonnyjohnny z/https://nus-sg.zoom.us/j/0123456789?pwd=ABCDEFGHIJKLMNOPDJFHISDFSDH`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+### Listing all persons : `clist`
+
+Shows a list of all persons in the contact list, with all of their details by default.
 
-### Listing all persons : `list`
+Format: `clist [e] [p] [a] [th] [z] [t]`
 
-Shows a list of all persons in the address book.
+* Returned list will always include names of all persons in the contact list.
+* When no optional fields are provided, e.g `clist`, the list will show all available details of all persons in the contact list.
+* When optional fields are provided, the list will only show the names of all persons in the contact list and the corresponding fields specified by the user.
+* More than one optional field can be provided.
+* The order of the optional fields does not matter. e.g both `clist e p` and `clist p e` will return a list of only the names, email addresses and phone numbers of all persons in the contact list.
+* If the specified field has no value for certain persons in the contact list, it will not show anything for that corresponding field for that particular person.
+
+Examples:
+* `clist` returns a list of all persons in the contact list with all the available details for each person.
+* `clist e p` returns a list of all persons in the contact list, showing only their names, email addresses and phone numbers (if available).
 
-Format: `list`
 
-### Editing a person : `edit`
+### Editing a person : `cedit`
 
-Edits an existing person in the address book.
+Edits an existing person in the contact list.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `cedit INDEX [n/NAME] [e/EMAIL] [p/PHONE] [a/ADDRESS] [th/TELEGRAM_HANDLE] [z/ZOOM] [dt/TAG_DELETED]… [t/TAG_ADDED]… `
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index must be a positive integer 1, 2, 3,…
```

Comment:
Should we just follow the original one in google docs? so bold "must be a positive integer" and put a spacing after comma `, ...`